packages/commons-ui-next/Link.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable jsx-a11y/anchor-has-content */
+import { Link as MuiLink } from "@mui/material";
+import clsx from "clsx";
+import { useRouter } from "next/router";
+import PropTypes from "prop-types";
+import React, { forwardRef } from "react";
+
+import NextComposed from "./NextComposed";
+
+// A styled version of the Next.js Link component:
+// https://nextjs.org/docs/#with-link
+const Link = forwardRef(function Link(props, ref) {
+  const {
+    href,
+    activeClassName = "active",
+    className: classNameProps,
+    naked,
+    ...other
+  } = props;
+
+  const router = useRouter();
+  const pathname = typeof href === "string" ? href : href?.pathname;
+  const className = clsx(classNameProps, {
+    [activeClassName]: router?.pathname === pathname && activeClassName,
+  });
+
+  const formattedHref = href;
+
+  const isRelative =
+    typeof formattedHref === "string" &&
+    (formattedHref.indexOf("/") === 0 || formattedHref.indexOf("#") === 0) &&
+    formattedHref.indexOf("//") !== 0;
+
+  if (!isRelative) {
+    const noProtocol = href?.startsWith("www.");
+    return (
+      <a
+        href={noProtocol ? `https://${href}` : href}
```

Comment:
```suggestion
      <MuiLink
        href={noProtocol ? `https://${href}` : href}
        target="_blank"
```
1. I think we'd still like to use Link styling,
2. External links should open on new tab.